pkg/git/poll/handler.go

```diff
@@ -3,17 +3,25 @@ package poll
 import (
 	"context"
 	"fmt"
-	"strings"
+	"time"
 
 	v1alpha1 "github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+	"github.com/rancher/fleet/pkg/git"
+	"github.com/reugn/go-quartz/quartz"
 
 	"github.com/go-logr/logr"
+	"github.com/rancher/fleet/internal/quartzretry"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	maxSchedulerOperationRetries = 3
+	defaultSyncInterval          = 15
```

Comment:
You can write this as `defaultSyncInterval = 15 * time.Second` which clarifies the scale of the number, you can then simplify the code below to remove the `* time.Second` when waiting?